cycles-ledger/src/main.rs

```diff
@@ -161,6 +161,7 @@ fn icrc1_metadata() -> Vec<(String, MetadataValue)> {
             "dfn:max_blocks_per_request",
             read_config(|config| config.max_blocks_per_request),
         ),
+        MV::entry("icrc1:logo", config::TOKEN_LOGO),
```

Comment:
Is there some test that verifies that the logo (or other metadata) is returned correctly?

Reply:
No, I tested it manually